identifiers/country-jp/single_member_constituencies.csv

```diff
@@ -0,0 +1,290 @@
+id,name
+ocd-division/country:jp/prefecture:01/ed:1,"Constituency 1, Hokkaido"

```

Comment:
These names are very repetitive. Wikipedia uses the style "Hokkaido 1", etc. https://en.wikipedia.org/wiki/Results_of_the_2017_Japanese_general_election